src/screens/Internal/index.tsx

```diff
@@ -1,10 +1,72 @@
 import React from 'react';
-import {View} from 'react-native';
+import {ResponseProps} from '../Home/types';
+import {
+    Container,
+    CharacterImage,
+    CharacterHeaderInfo,
+    CharacterInfo,
+    CharacterInfoConfig,
+    CharacterName,
+} from './styles';
+import Text from '../../components/Text';
+import Separator from '../../components/Separator';
+import Icon from '../../components/Icon';
+import HeaderOptions from '../../components/HeaderOptions';
+import {Pressable} from 'react-native';
+import Row from '../../components/Row';
+import {useNavigation} from '@react-navigation/native';
 
-// import { Container } from './styles';
+const Internal = ({route}: any) => {
+    const {item}: ResponseProps = route.params;
 
-const Internal: React.FC = () => {
-    return <View />;
+    const {goBack} = useNavigation();
+
+    return (
+        <Container>
+            <CharacterImage source={{uri: item?.image}}>
+                <HeaderOptions
+                    left={
+                        <Pressable onPress={goBack}>
+                            <Row>
+                                <Icon icon="arrowLeft" size={20} />
+                                <Separator width={10} />
+                                <Text size={20}>Back</Text>
+                            </Row>
+                        </Pressable>
+                    }
+                />
+            </CharacterImage>
+
+            <CharacterInfoConfig>
+                <CharacterHeaderInfo>
+                    <CharacterName isBold size={30}>
```

Comment:
move those text props to an `attrs({ })` prop in the `styles.ts`

Reply:
move every styled component with props usage to the `attrs` prop in the styles file

Reply:
resolved in commit https://github.com/IagoWag/rickandmorty/commit/a71461893b6fea906309d9e3507e0c49e0d139f5